uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/components/InputFieldModifierWithFocusChangedAndScrollingLogic.kt

```diff
@@ -0,0 +1,55 @@
+package tech.dojo.pay.uisdk.presentation.components
+
+import androidx.compose.foundation.ScrollState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.layout.onGloballyPositioned
+import androidx.compose.ui.layout.positionInRoot
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlin.math.roundToInt
+
+@Composable
+fun InputFieldModifierWithFocusChangedAndScrollingLogic(
```

Comment:
This is giving you a warning because composable functions that returns something must start with lower case.
As this is a modifier, I would suggest making an extension function of Modifier. Also a slight change in the name
```suggestion
internal fun Modifier.autoScrollableInputFieldOnFocusChange(...)
```